### DISCUSSION TITLE
N64 transfer pack

### DISCUSSION THREAD

**Discussion by Hermit33** [mupen64plus-ae/mupen64plus-ae#911] [Ideas]
Hey I just have a quick idea is there a way to make it link to another emulator like the Nintendo 64 emulator links up with the Game Boy emulator in a way to be able to transfer Pokemon from red and blue to Pokemon stadium 1 and 2

**Comment by fzurita** [comment]
Hello, this is already possible, see here for a guide: http://www.paulscode.com/forum/index.php?topic=15903.0
